apps/accounts/risk_handlers.py

```diff
@@ -193,5 +193,5 @@ def handle_change_password_add(self):
         (
             GatheredAccount.objects
             .filter(asset=self.asset, username=self.username)
-            .update(present=True)
+            .update(present=True, status=ConfirmOrIgnore.confirmed)
         )
```

Comment:
There does not seem to be any significant differences between this line:

```
        GatheredAccount.objects.filter(asset=self.asset, username=self.username).update(present=True)  
```

and the version provided in your comment (which also updates `status` but only adds it if an existing account with that username exists):

```
        GatheredAccount.objects.update(status=ConfirmOrIgnore.confirmed)
```